components/ResidentPage/Mapping.tsx

```diff
@@ -51,6 +51,8 @@ const Mapping = ({ socialCareId }: Props): React.ReactElement | null => {
 
   if (!data) return null;
 
+  if (!data.address?.postcode && !data.address?.address) return null; // handle resident with no address set
```

Comment:
Not seeing an accompanying test in `./components/ResidentPage/Mapping.spec.tsx` around this change.